sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldNotBeIgnored.NUnit.cs

```diff
@@ -1,11 +1,14 @@
 ﻿using System;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using NUnit.Framework;
+using MSTest = Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace Tests.Diagnostics
 {
-    class NUnitClass
+    [TestFixture]
+    public class NUnitClass
     {
+        // False positive: using an MSTest Ignore against an NUnit test does nothing,
+        // but we still raise an issue.
```

Comment:
It's unlikely anyone would do this in practice so I don't think it's worth worrying about.